src/placeos-driver/storage/edge-storage.cr

```diff
@@ -2,41 +2,38 @@ require "../storage"
 require "../protocol"
 
 class PlaceOS::Driver::EdgeStorage < PlaceOS::Driver::Storage
-  def initialize(@id : String, @prefix = DEFAULT_PREFIX)
-    super()
-    @hash_key = "#{prefix}/#{@id}"
+  private getter hash : Hash(String, String) = {} of String => String
+  forward_missing_to hash
```

Comment:
Is forwarding everything desirable? What happens when `#delete(key, &)` or `#shift` is called?